robocorp-code/src/robocorp_code/robocorp_language_server.py

```diff
@@ -990,55 +1015,41 @@ def _get_locators_json_info(
             }
 
         path = Path(params["robotYaml"])
-        locators_json = path.parent / "locators.json"
         locators_json_info: List[LocatorEntryInfoDict] = []
         locator: Locator
-        if locators_json.exists():
+        db, locators_json = None, None
+        try:
+            action_result: ActionResultDictLocatorsJson = self._load_locators_db(path)
+            if action_result["success"]:
+                db, locators_json = action_result["result"]
+            else:
+                raise Exception(action_result["message"])
+
             with locators_json.open("r") as stream:
                 contents = stream.read()
             content_lines = contents.splitlines()
 
-            try:
-                db = LocatorsDatabase(str(locators_json))
-                db.load()
-                if not db.locators.items():
-                    error = db.error
-                    if not isinstance(error, str):
-                        if isinstance(error, tuple) and len(error) == 2:
-                            try:
-                                error = error[0] % error[1]
-                            except:
-                                error = str(error)
-                        else:
-                            error = str(error)
-
-                    return {"success": False, "message": error, "result": None}
-
-                for name, locator in db.locators.items():
-                    as_dict = locator.to_dict()
-                    line, col = self._get_line_col(name, content_lines)
-                    locators_json_info.append(
-                        {
-                            "name": name,
-                            "line": line,
-                            "column": col,
-                            "type": as_dict["type"],
-                            "filePath": str(locators_json),
-                        }
-                    )
-            except Exception as e:
-                log.exception(f"Error loading locators from: {locators_json}")
-                return {"success": False, "message": str(e), "result": None}
+            for name, locator in db.locators.items():
+                as_dict = locator.to_dict()
+                line, col = self._get_line_col(name, content_lines)
+                locators_json_info.append(
+                    {
+                        "name": name,
+                        "line": line,
+                        "column": col,
+                        "type": as_dict["type"],
+                        "filePath": str(locators_json),
+                    }
+                )
+        except Exception as e:
+            return {"success": False, "message": str(e), "result": None}
```

Comment:
Exceptions should be logged (i.e.: `log.exception("Error message")`).

Reply:
Added log.exception()